src/main/java/org/osiam/storage/query/QueryFilterParser.java

```diff
@@ -38,10 +36,15 @@ public ParseTree getParseTree(String filter) {
         if (Strings.isNullOrEmpty(filter)) {
             return null;
         }
+        OsiamAntlrErrorListener errorListener = new OsiamAntlrErrorListener();
 
         LogicalOperatorRulesLexer lexer = new LogicalOperatorRulesLexer(new ANTLRInputStream(filter));
+        lexer.removeErrorListeners();
+        lexer.addErrorListener(errorListener);
```

Comment:
This needs some explanation. Has it to do with https://github.com/osiam/osiam/pull/125/files#r51790934?